org.srb2.SRB2Kart.yaml

```diff
@@ -11,20 +11,47 @@ finish-args:
   - --socket=wayland
   - --device=all
   - --persist=.srb2kart
+  - --filesystem=xdg-run/app/com.discordapp.Discord:create
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /lib/cmake
+  - /man
+  - /share/doc
```

Comment:
It is better practice to have individual modules do their own cleanup.

Reply:
that should be better